_studio/mfx_lib/encode_hw/h265/src/mfx_h265_encode_hw_ddi.cpp

```diff
@@ -103,6 +107,8 @@ mfxStatus HardcodeCaps(MFX_ENCODE_CAPS_HEVC& caps, VideoCORE* core)
         caps.ddi_caps.NegativeQPSupport = 0;
     else
         caps.ddi_caps.NegativeQPSupport = 1; // driver should set it for Gen11+ VME only
+		
+    caps.PSliceSupport = (IsOn(par.mfx.LowPower) || platform > MFX_HW_ICL) ? 0 : 1;
```

Comment:
What is libva bug number to properly enable caps?

Reply:
I believe there no such issue yet. Let us involve  @gkrivor, @saosipov, @fzhar, @eshiryae, @nastsokolova and start to form a proposal.